docs/pages/concepts/offloading/resource-reflection.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Resource reflection
+weight: 4
+---
+
+In addition to Node and Pods lifecycle handling, the Liqo virtual kubelet implements a feature we call *resource reflection*.
+It is responsible for the propagation and synchronization of selected control plane information into remote clusters, to enable the seamless execution of offloaded Pods. Currently, the virtual kubelet supports the reflection of the following Kubernetes resources:
+
+* `Services`, abstracting the access to workloads spread across multiple clusters, while enabling the usage of standard DNS discovery mechanisms.
```

Comment:
1) I wonder whether we should also say _how_ are reflected (e.g., are reflected as they are? Are modified?)
2) I would write also *which* resources are reflected. I presume only the ones that belong to a reflected namespace, bit I would say this explicitly.